services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -678,9 +678,9 @@ testCreateUserTimeout = do
       tryquery (filterBy "externalId" $ fromEmail email)
 
     waitUserExpiration = do
-      timeoutSecs <- asks $ fromMaybe 5 . brigSettingsTeamInvitationTimeout . _teOpts
+      timeoutSecs <- asks (cfgBrigSettingsTeamInvitationTimeout . view teTstOpts)
       Control.Exception.assert (timeoutSecs < 30) $ do
-        threadDelay $ (timeoutSecs + 2) * 1_000_000
+        threadDelay $ (timeoutSecs + 1) * 1_000_000
```

Comment:
I reverted +2 seconds from #1333 back, because it was working fine with +1 before.